### PR TITLE
HIVE-26158: TRANSLATED_TO_EXTERNAL partition tables cannot query partition data after rename table

### PR DESCRIPTION
**Pull request by kgyrtkirk**

### CODE REVIEW

File: standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/utils/MetaStoreUtils.java [hunk 1]
```diff
@@ -244,6 +244,13 @@ public static boolean isExternalTable(Table table) {
     return isExternal(params);
   }
 
+  public static boolean isTranslatedToExternalTable(Table table) {
+    Map<String, String> p = table.getParameters();
```
Comment (saihemanth-cloudera) [inline standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/utils/MetaStoreUtils.java:248]: Can you change the name of this variable 'p' to something more meaningful like tblProperties?

File: ql/src/test/queries/clientpositive/translated_external_rename3.q [hunk 2]
```diff
@@ -1,26 +1,25 @@
 set metastore.metadata.transformer.class=org.apache.hadoop.hive.metastore.MetastoreDefaultTransformer;
-set metastore.metadata.transformer.location.mode=force;
```
Comment (saihemanth-cloudera) [inline ql/src/test/queries/clientpositive/translated_external_rename3.q:2]: this is also working in force mode also. Can you please give some info about why you had to change it to seqprefix?

Comment (kgyrtkirk) [reply ql/src/test/queries/clientpositive/translated_external_rename3.q:2]: this was a mistake - I've clobbered this existing test too much; I've moved the new testcase into a new qfile.

the location mode shouldn't matter for this test at all.

**Review by saihemanth-cloudera — APPROVED** [review]
Changes look good to me. +1

**Pull request merged**